### PR TITLE
Add section for maven-runner settings

### PR DESCRIPTION
**Pull request by martin-strecker-sonarsource**
It is necessary to delegate the build to maven in some projects. This PR adds a section where to find the options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,3 +164,9 @@ IDEA must also be manually configured for imports : `Preferences/Settings > Edit
 Then go to `Preferences/Settings > Editor > General > Auto import` and check "Optimize imports on the fly" for Java:
 
 ![image](https://user-images.githubusercontent.com/50145663/135304815-0b51f1da-16d4-4fc7-a278-a13067e78d7c.png)
+
+## (Optional) Built Configuration for Intellij
+
+If you experience problems with the build-in build system (*Build* ➡️ *Rebuild Project*), you should delegate the build to maven. Make sure, to skip tests during the build:
```
Comment (pavel-mikula-sonarsource) [inline README.md:170]: This toolset is used also with other systems like Gradle AFAICT. And I have no idea how that behaves

File: README.md [hunk 2]
```diff
@@ -164,3 +164,9 @@ IDEA must also be manually configured for imports : `Preferences/Settings > Edit
 Then go to `Preferences/Settings > Editor > General > Auto import` and check "Optimize imports on the fly" for Java:
 
 ![image](https://user-images.githubusercontent.com/50145663/135304815-0b51f1da-16d4-4fc7-a278-a13067e78d7c.png)
+
+## (Optional) Built Configuration for Intellij
```
Comment (johann-beleites-sonarsource) [inline README.md:168]: I think we can make the section title more precise here. E.g.
```suggestion
## (Optional) Configuring IntelliJ to Build using Maven
```

File: README.md (hunk 1 above)
Comment (johann-beleites-sonarsource) [inline README.md:170]: Can you give an example of the problems you may experience that warrant enabling this feature? At the moment the 'why' to using (or not using) this functionality seems not 100% clear to me.

**Review by johann-beleites-sonarsource — COMMENTED** [review]
Nice addition! I have a couple of questions/suggestions

File: README.md (hunk 1 above)
Comment (johann-beleites-sonarsource) [inline README.md:170]:
```suggestion
If you experience problems with the built-in build system (*Build* ➡️ *Rebuild Project*), you should delegate the build to maven. Make sure to skip tests during the build:
```

File: README.md (hunk 2 above)
Comment (martin-strecker-sonarsource) [reply README.md:168]: I was considering this too but looking at the general structure of the document, it seemed more appropriate to keep the title of the section neutral:

```markdown
## SonarLint
## Git
## Eclipse Configuration
## Code Style Configuration for Intellij
## (Optional) Build Configuration for Intellij
```

Another option would be to make this a sub-section of `## Intellij Configuration`:

```markdown
## SonarLint
## Git
## Eclipse Configuration
## Intellij Configuration
### Code Style 
### (Optional) Build Configuration
```

File: README.md (hunk 1 above)
Comment (martin-strecker-sonarsource) [reply README.md]: I gave some examples and added a link to the maven documentation. They mention the circumstances when a maven build is preferable over an "IDEA build" there.

**Review by johann-beleites-sonarsource — APPROVED** [review]

**Pull request merged**